docstring_parser/parser.py

```diff
@@ -1,13 +1,18 @@
 """The main parsing routine."""
 
 from docstring_parser import epydoc, google, numpydoc, rest
-from docstring_parser.common import Docstring, DocstringStyle, ParseError
+from docstring_parser.common import (
+    Docstring,
+    DocstringStyle,
+    ParseError,
+    RenderingStyle,
+)
 
 STYLES = {
-    DocstringStyle.rest: rest.parse,
-    DocstringStyle.google: google.parse,
-    DocstringStyle.numpydoc: numpydoc.parse,
-    DocstringStyle.epydoc: epydoc.parse,
+    DocstringStyle.rest: rest,
+    DocstringStyle.google: google,
+    DocstringStyle.numpydoc: numpydoc,
+    DocstringStyle.epydoc: epydoc,
```

Comment:
Let's rename this variable to `_STYLE_MAP` to make it clear that we break backwards compatibility here

Reply:
ok.